src/video-grid/VideoTileSettingsModal.tsx

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2022 Matrix.org Foundation C.I.C.
```

Comment:
This should be New Vector Ltd instead - the copyright in most other files is currently wrong